Mobl_ios_Callisto-master/MealTrackingPilotTests/Mocks/MockPilotAPIClient.swift

```diff
@@ -8,27 +8,26 @@
 
 import Foundation
 import Realm
-import Intrepid
 @testable import MealTrackingPilot
 
 enum MockPilotAPIClientError: Error {
     case errorFromServer
 }
 
 class MockPilotAPIClient: PilotAPIClient {
-    var mockCreateMealEventResponse: Result<RealmMealEvent> = .success(RealmMealEvent())
-    var mockCreateImageURLResponse: Result<String> = .success("")
-    var mockCreateLocationResponse: Result<Void> = .success(())
+    var mockCreateMealEventResponse: Result<RealmMealEvent,Error> = .success(RealmMealEvent())
+    var mockCreateImageURLResponse: Result<String,Error> = .success("")
+    var mockCreateLocationResponse: Result<Void,Error> = .success(())
 
-    override func createMeal(_ meal: Meal, completion: ((Result<RealmMeal>) -> Void)?) {
+    override func createMeal(_ meal: Meal, completion: ((Result<RealmMeal,Error>) -> Void)?) {
```

Comment:
Change ((Result<RealmMeal,Error>) -> Void) to relamMealCompletion

Reply:
Done